src/main/java/com/w6/data/dao/event/EventServiceImpl.java

```diff
@@ -42,7 +42,48 @@ public Event save(Event event) {
     }
 
     @Override
-    public long count() {
-        return eventRepository.count();
+    public List<Event> findAll() {
+        List<Event> result = new ArrayList<>();
+
+        Iterable<Event> events = eventRepository.findAll();
```

Comment:
Nice exercice on java rewrite this function into 1 line using 
http://winterbe.com/posts/2014/07/31/java8-stream-tutorial-examples/